app/controllers/animals_controller.rb

```diff
@@ -0,0 +1,7 @@
+class AnimalsController < ApplicationController
```

Comment:
This is not an API controller? Take a look at my other comments below where you have both a React component for the index and an ERB template. From this controller it looks like you intend to use the ERB template, in which case you can delete the component, or vice versa. In either case, your view (template or component) should be set up to display the animals data